CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ##### Enhancements
 
+* [8848](https://github.com/grafana/loki/pull/8848) **dannykopping**: Ruler: rule evaluation jittter.
```

Comment:
nit:
(I think it is a better reading for end users)
```suggestion
* [8848](https://github.com/grafana/loki/pull/8848) **dannykopping**: Add new ruler `evaluation_jitter` configuration.
```

Reply:
We try and keep the subsystem name at the beginning to make it easier to identify which changes someone might be interested in